### DISCUSSION TITLE
How to get "Public Sponsor" badge?

### DISCUSSION THREAD

**Discussion by bojanterzic529** [community/community#30099] [Other Feature Feedback, Questions, & Ideas] [answered]
I am gathering github badges, but I didn't get "Public Sponsor" badge.
If you know how I to do it, please let me know

**Comment by flextra19** [comment, marked as answer]
https://github.com/sponsors/explore
Choose someone and hit the 'Sponsor' button, then you can pay for that person and become a sponsor.

**Reply by bojanterzic529** [reply]
hmmm,  let me check

**Reply by bojanterzic529** [reply]
yes, that's right
I got "Public sponser" badge.
It's super easy.
thanks
